ledger-api/grpc-definitions/BUILD.bazel

```diff
@@ -259,6 +265,19 @@ pom_file(
     target = ":ledger-api-scalapb",
 )
 
+# Create empty Scaladoc JAR for uploading to Maven Central
+scaladoc_jar(
+    name = "ledger-api-scalapb_scaladoc",
+    srcs = [],
+    deps = [],
+) if is_windows == False else None
+
+# Create empty Sources JAR for uploading to Maven Central
+scala_source_jar(
+    name = "ledger-api-scalapb_src",
```

Comment:
Please modify the `scala_source_jar` rule to allow providing a `srcjar` label.

Reply:
Implemented by #1602 and #1611